### DISCUSSION TITLE
Asus Update don't show up on G-Helper Updates

### DISCUSSION THREAD

**Discussion by lostfocus94** [seerge/g-helper#2791] [Q&A]
Hi, I seems to have a problem in my G-Helper which don't show anything on Updates tab. Is there anything I can do to fix it?
![Screenshot 2024-06-29 142536](https://github.com/seerge/g-helper/assets/15966441/a7453422-a9ad-4b11-9df1-05ac4afc0676)

**Comment by lostfocus94** [comment]
Closed, turns out I just need to wait for a little while, sorry

**Comment by seerge** [comment]
@lostfocus94 it checks for updates online from asus website. Did you block internet connection for the app?

**Reply by lostfocus94** [reply]
It's fixed now, I just have to wait it to load up apparently. And I also made sure that the firewalls allow G-Helper to access internet.

![image](https://github.com/seerge/g-helper/assets/15966441/84ab9401-4d1d-4104-a9d2-56774b9cc321)
